client/src/index.jsx

```diff
@@ -117,6 +135,18 @@ class App extends React.Component {
 		return animalList;
 	}
 
+	showLogin() {
+		return <Login onLogin={this.updateUserState} return={this.viewHomepage} />
+	}
+
+	updateUserState() {
+		console.log("App is updating user state");
+		const newUser = localStorage.getItem('user');
```

Comment:
As far as I can tell, this is just going to be the username. Won't you also need the user ID?

Reply:
Updated name to 'user' from newUser

Reply:
Updated localStorage to save entire user info (minus password)